### PR TITLE
WIP: Version Control in Bintray

### PR DESCRIPTION
**Pull request by panpan0000**
detail in https://groups.google.com/forum/#!topic/rackhd/fEtZVKjKmZ0

relevant  PR: 
https://github.com/RackHD/on-dhcp-proxy/pull/58
https://github.com/RackHD/on-skupack/pull/51

### CODE REVIEW

**Comment by rackhdmirror** [comment]
@RackHD/corecommitters please review.

**Comment by panpan0000** [comment]
@ytjohn   @RackHD/doccommitters

**Comment by ytjohn** [comment]
I'd like to review this a bit more today. The name scheme for a package is
pretty important. The "latest" package is determined by alphabetical
ordering. For example "stable" is alphabetically more recent than "master"
or "devel". "1.1.1-201608" is older than "201605".

**Comment by panpan0000** [comment]
more discussion in Google Group:
https://groups.google.com/forum/#!topic/rackhd/fEtZVKjKmZ0

File: .travis.yml [hunk 1]
```diff
@@ -20,8 +20,8 @@ after_success:
  - ./extra/make-testcoveralls.sh
 
 before_deploy:
+ - ./gen_bintray_cfg.bash "${TRAVIS_BRANCH}"
```
Comment (changev) [inline .travis.yml]: Can I have some information about where does the TRAVIS_BRANCH come from?

Comment (panpan0000) [inline .travis.yml]: generated by Travis CI as a runtime ENV VAR:
https://docs.travis-ci.com/user/environment-variables/

**Pull request closed without merging**